python/src/IntervalMesher_doc.i.in

```diff
@@ -28,9 +28,10 @@ interval : :class:`~openturns.Interval`
     The interval to mesh, of dimension equal to the dimension
     of `discretization`.
 diamond : bool
-    A flag telling if one want the usual mesh (rectangular cells cut along a
-    diagonal in dimension 2) or a diamond like mesh (rectangular cells cut
-    into 4 triangles with the center of the cell as shared vertex).
+    Set to True to compute rectangular cells cut into 4 triangles with the center of
```

Comment:
This is the contrary..... I propose to write: 

A flag telling if the mesh is composed of diamond like mesh  (rectangular cells cut  into 4 triangles with the center of the cell as shared vertex).
If False, the mesh is composed of rectangular cells cut along a  diagonal in dimension 2.
Default value is False.